app/controllers/api/grades_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+module Api
+  class GradesController < ApiController
```

Comment:
I thought about this some more and I think it should be a separate controller.  The mobile client will often want to pull in all grades it can, regardless of students or lessons. This makes me think that they should be a top level resource and have a dedicated controller.

The reason why the web interface doesn't use this is because the web client will only look at the grades in context of a specific student and lesson.